crates/db_views/src/custom_emoji_view.rs

```diff
@@ -57,6 +57,42 @@ impl CustomEmojiView {
     Ok(CustomEmojiView::from_tuple_to_vec(emojis))
   }
 
+  pub async fn list(
+    pool: &mut DbPool<'_>,
+    for_local_site_id: LocalSiteId,
+    category: &Option<String>,
+    page: Option<i64>,
+    limit: Option<i64>,
+  ) -> Result<Vec<Self>, Error> {
+    let conn = &mut get_conn(pool).await?;
+    let (limit, offset) = limit_and_offset(page, limit)?;
+    let mut query = custom_emoji::table
+      .left_join(
+        custom_emoji_keyword::table.on(custom_emoji_keyword::custom_emoji_id.eq(custom_emoji::id)),
+      )
+      .into_boxed();
+
+    query = query.filter(custom_emoji::local_site_id.eq(for_local_site_id));
```

Comment:
This is unnecessary because all emojis have the same local site id. Honestly you can remove the `local_site_id` column, it doesnt seem to have any use.

Reply:
I made a similar point regarding taglines.